plugins/Live/templates/_dataTableViz_visitorLog.twig

```diff
@@ -15,11 +15,14 @@
                 <ul class="details">
                     <li>{{ 'DevicesDetection_ColumnBrowser'|translate }}: {{ visitor.getColumn('browser') }}</li>
                     <li>{{ 'DevicesDetection_BrowserEngine'|translate }}: {{ visitor.getColumn('browserFamily') }}</li>
-                    {% if visitor.getColumn('pluginsIcons')|length > 0 %}<li>{{ 'General_Plugins'|translate }}:
-                        {% for pluginIcon in visitor.getColumn('pluginsIcons') %}
-                            <img src="{{ pluginIcon.pluginIcon }}" title="{{ pluginIcon.pluginName|capitalize(true) }}" alt="{{ pluginIcon.pluginName|capitalize(true) }}"/>
-                        {% endfor %}
-                    </li>{% endif %}
+                    {% if visitor.getColumn('pluginsIcons')|length > 0 %}
+                        <li>
+                            {{ 'General_Plugins'|translate }}:
+                            {% for pluginIcon in visitor.getColumn('pluginsIcons') %}
+                                <img src="{{ pluginIcon.pluginIcon }}" alt="{{ pluginIcon.pluginName|capitalize(true) }}"/>
```

Comment:
On purpose, there is already the `alt` for screen readers. And since it's in a tooltip, it's impossible to hover (and then show another tooltip in a tooltip), so I've removed it.